packages/fast-development-site-react/src/components/site/configuration-panel.tsx

```diff
@@ -0,0 +1,213 @@
+import * as React from "react";
+import { toPx } from "@microsoft/fast-jss-utilities";
+import { IDevSiteDesignSystem } from "../design-system";
+import Form from "@microsoft/fast-form-generator-react";
+import { Link, withRouter } from "react-router-dom";
+import { RouteComponentProps } from "react-router";
+import { ComponentViewTypes } from "./component-view";
+import { glyphBuildingblocks, glyphExamples, glyphPage } from "@microsoft/fast-glyphs-msft";
+import ComponentViewToggle from "./component-view-toggle";
+import manageJss, { ComponentStyles, ICSSRules, IJSSManagerProps, IManagedClasses } from "@microsoft/fast-jss-manager-react";
+
+export enum TabType {
+    presets = "Presets"
+};
+
+export interface IConfigurationPanelProps {
+    // onToggleView: () => void;
```

Comment:
What are your plans for the commented out lines?  Is there future work to do?  If so we should create a single todo statement with work issue number and description.  Otherwise, I recommend deleting the commented out code.

Reply:
We should remove commented out code prior to check-in.